src/solve.jl

```diff
@@ -131,19 +134,16 @@ function _solve(cp::ControlledPlant,
     FT = Flowpipe{NT, RT, Vector{RT}}
     out = Vector{FT}(undef, NSAMPLES)
 
+    X = nothing
```

Comment:
I will soon propose a reorderung of the code that will make this step unnecessary.

Reply:
I created an issue (#178).